hooks/useGovernanceAssets.ts

```diff
@@ -169,6 +169,10 @@ export default function useGovernanceAssets() {
     [PackageEnum.Common]: {
       name: 'Common',
     },
+    [PackageEnum.Dual]: {
+      name: 'Dual',
+      image: '/img/dual-logo.png',
+    },
```

Comment:
Should we use full name?

```suggestion
    [PackageEnum.DualFinance]: {
      name: 'Dual Finance',
      image: '/img/dual-logo.png',
    },
```

Before:
<img width="541" alt="Screenshot 2022-11-30 at 17 30 11" src="https://user-images.githubusercontent.com/22965416/204809217-a512bd8a-aa6f-4172-9d0c-f8c5cb20985a.png">

After:
<img width="530" alt="Screenshot 2022-11-30 at 17 30 53" src="https://user-images.githubusercontent.com/22965416/204809264-554d6749-2c15-4679-9ee1-1fc5d7530da9.png">